packages/components/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import * as ToastPrimitive from "@radix-ui/react-toast";
+import { Anchor } from "../Anchor";
+import { Box } from "../../primitives/Box";
+import { Text } from "../../primitives/Text";
+import { Icon } from "../Icon";
+import type { ToastVariants } from "./types";
+
+export interface ToastProps
+  extends Omit<React.ComponentProps<typeof ToastPrimitive.Root>, "asChild"> {
+  /** The text used if an Anchor is used in the toast. */
+  anchorText?: string;
+  /** The href used if an Anchor is used in the toast. */
+  anchorHref?: string;
```

Comment:
Can we update the type so that anchorText and anchorHref are both needed or both not needed? Right now, it's possible to provide an anchorText but no anchorHref which would result in nothing appearing.

Reply:
Yep. I added a conditional type so the cta props are only required if `hasCta` is `true`.

Reply:
Is there a need for `hasCta` now that you have the conditional type?